cmd/cstor-volume-mgmt/controller/volume-controller/handler_test.go

```diff
@@ -26,14 +26,21 @@ import (
 	openebsFakeClientset "github.com/openebs/maya/pkg/client/generated/clientset/versioned/fake"
 	informers "github.com/openebs/maya/pkg/client/generated/informers/externalversions"
 	"github.com/openebs/maya/pkg/client/k8s"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	kubeinformers "k8s.io/client-go/informers"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/kubernetes/fake"
 )
 
+//TODO: Move to some common function
+func StrToQuantity(capacity string) resource.Quantity {
```

Comment:
let us make this private & put mock or test as suffix to avoid being used in source code

Reply:
renamed to `fakeStrToQuantity`